### DISCUSSION TITLE
Shipping Category Condition "Ignore"

### DISCUSSION THREAD

**Discussion by JustJarethB** [craftcms/commerce#2357] [Ideas]
### Description
Request for the `Allow` `Disallow` `Require` conditions to be complimented with `Ignore` where the rule can be used with said items in the cart, but their value is not considered in the validity calculations.

### Example Use Case
2 Categories, `General` and `Downloads` where `Downloads` does not require shipping. We wish to allow them to be in the cart with all other shippable products, without affecting the shipping calculations of the other items.

**Comment by bradstemke** [comment]
We are experiencing an issue similar to this. Have not yet found a solution for it. 

Just so I understand if I should follow this issue, you are also reporting that items in the cart with shipping combined with an item that is set to no shipping, that item overrides the shipping for the cart and there is no shipping applied, yeah?

**Comment by JustJarethB** [comment]
@bradstemke It depends on the exact configuration but that sounds like a possibility? My personal issue is more like shipping over $5 and an item that costs $2, plus an item that costs $10 but should be free shipping anyway, the $10 should not count towards the $5 total
